feature-detects/inputsearchevent.js

```diff
@@ -2,6 +2,7 @@
 {
   "name": "input[search] search event",
   "property": "inputsearchevent",
+  "caniuse": "input-search",
```

Comment:
this looks like the element in the caniuse database while  modernizr tests if there a spoecial search event.. better remove this